modules/ztp-removing-content-from-managed-clusters.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-configuring-managed-clusters-policies.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-removing-content-from-managed-clusters_{context}"]
+= Changing applied managed cluster CRs using policies
+
+You can remove content from a custom resource (CR) deployed to a managed cluster through a policy.
+
+By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have a `complianceType` set to `musthave`.
+In this mode, when you remove a field from a CR, the value is removed from the policy but is not removed from the CR on the managed cluster.
+A `musthave` policy without the removed field is still compliant because the CR on the managed cluster has all the specified content.
```

Comment:
Are we discussing a specific removed fields or any removed default fields? I think clarification is necessary here.

Reply:
I think the removed field is the field removed in the previous sentence??

Reply:
Okay, so I think having the sentences in this order is a bit confusing. Does it help if I switch it around like this?

> By default, all `Policy` CRs created from a `PolicyGenTemplate` CR have the `complianceType` field set to `musthave`.
> A `musthave` policy without the removed content is still compliant because the CR on the managed cluster has all the specified content.
> In this configuration, when you remove content from a CR, {cgu-operator} removes it from the policy but the content is not removed from the CR on the managed cluster.

And then talking about why the user needs the other configuration:
> With the `complianceType` field to `mustonlyhave`, the policy ensures that the CR on the cluster is an exact match of what is specified in the policy.

Reply:
Looks good. Slight nitpick with:

> With this configuration, when you remove content from a CR, the {cgu-operator} removes the content from the policy, but the content is not removed from the CR on the managed cluster.